.github/workflows/jira.yaml

```diff
@@ -58,8 +63,11 @@ jobs:
         else
           echo "type=ISS" >> $GITHUB_OUTPUT
         fi
+        if [[ "${{ github.triggering_actor }}" == "dependabot[bot]" ]] && [[ "${{ inputs.ignore_dependabot }}" == true ]]; then
+          echo "short_circuit=true" >> $GITHUB_OUTPUT
+        fi
     - name: Create ticket
-      if: github.event.action == 'opened' || github.event.action == 'created'
+      if: steps.preprocess.outputs.short_circuit != true && (github.event.action == 'opened' || github.event.action == 'created')
```

Comment:
All good to make it systematic. Dependabot PRs already add the dependency label so might as well use it instead of introducing a specific dependabot label.